requirements.txt

```diff
@@ -296,6 +296,7 @@ multidict==6.0.2
 mypy-extensions==1.0.0
     # via black
 natsort==8.4.0
+nanoid==2.0.0
```

Comment:
we use requirements.in to add reqs, and pip-compile to generate this file!

Reply:
fixed now